test/blackbox-tests/test-cases/coq/coqtop/coqtop-recomp.t

```diff
@@ -22,19 +22,19 @@ https://github.com/ocaml/dune/pull/5457#issuecomment-1084161587).
         coqdep dir/bar.v.d
         coqdep dir/foo.v.d
           coqc dir/foo.{glob,vo}
-  -topfile $TESTCASE_ROOT/_build/default/dir/bar.v -q -w -deprecated-native-compiler-option -w -native-compiler-disabled -native-compiler ondemand -R $TESTCASE_ROOT/_build/default/dir basic
+  -topfile $TESTCASE_ROOT/_build/default/dir/bar.v -w -deprecated-native-compiler-option -w -native-compiler-disabled -native-compiler ondemand -R $TESTCASE_ROOT/_build/default/dir basic
```

Comment:
This is a bug too, all `coqc` invocations should have `-q` in any case.

Reply:
These are not the flags passed to coqc but to coqtop only.

Reply:
Mmm, I got confused, that line comes after a `coqc` call? Never mind if I misunderstood the log.